ztp/source-crs/PtpOperatorConfigForEvent.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpOperatorConfig
+metadata:
+  name: default
```

PtpOperatorConfig is a Namespace scoped CR. Can you add
```suggestion
  name: default
  namespace: openshift-ptp
```

---

done